references/oidc-mock/README.md

```diff
@@ -24,7 +24,8 @@ Standard endpoints are exposed, like:
     export APIGEE_PASS=xxx
 
     export PATH="$PATH:$SCRIPTPATH/../../tools/apigee-sackmesser/bin"
-    sackmesser deploy --apigeeapi -o "$APIGEE_ORG" -e "$APIGEE_ENV" -u "$APIGEE_USER" -p "$APIGEE_PASS"
+    sackmesser deploy --apigeeapi -o "$APIGEE_ORG" -e "$APIGEE_ENV"
+    -u "$APIGEE_USER" -p "$APIGEE_PASS"
```

Comment:
this newline here seems to break the command - it might also cause problems for people copying/pasting. Could we put this on the same line as the others in line 27?

Reply:
ok Ozan